fdbcli/BlobKeyCommand.actor.cpp

```diff
@@ -56,14 +60,19 @@ ACTOR Future<bool> printBlobHistory(Database db, Key key, Optional<Version> vers
 			wait(tr.onError(e));
 		}
 	}
+	if (tenantEntry.present()) {
+		activeGranule = activeGranule.withPrefix(tenantEntry.get().prefix);
```

Comment:
Since we have the `Tenant` object here, we don't need to pass in the tenant entry:

```suggestion
	if (tenant.present()) {
		activeGranule = activeGranule.withPrefix(tenant.get()->prefix());
```